tutorials/installation.md

```diff
@@ -0,0 +1,67 @@
+# napari installation tutorial
+
+Welcome to the **napari** installation tutorial!
+
+This tutorial will teach you how to do a clean install of **napari**. It is aimed at people that want to use **napari**. For people interested in contributing to **napari** please check our [Contributing Guidelines](../CONTRIBUTING.md) for more advanced installation procedures. At the end of the tutorial you should have **napari** successfully installed on your computer and be able to make the **napari** viewer appear.
+
+## instalation
+
+**napari** can be installed on most Mac OS X and Linux systems with Python 3.6 or 3.7. We're currently working on improving Windows support. There are three different ways to install napari.
```

Comment:
macOS. As we've seen, we can't really install on Mac OS X. =P

Reply:
fixed